components/aws/sagemaker/tests/integration_tests/component_tests/test_batch_transform_component.py

```diff
@@ -82,4 +83,9 @@ def test_transform_job(
     )
     assert s3_utils.check_object_exists(s3_client, s3_data_bucket, file_key)
 
+    error_message = 'Error in fetching CloudWatch logs for SageMaker job'
```

Comment:
This is still not referencing the error message you defined in `_utils.py`. That means our integration tests will break if we change the message.